### PR TITLE
feat: Implement example service using ASP.NET Core

### PR DESCRIPTION
**Pull request by agukasm**

### CODE REVIEW

File: src/WebEid.AspNetCore.Example/Certificates/CertificateLoader.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿namespace WebEid.AspNetCore.Example.Certificates
+{
+	using System.Collections.Generic;
+	using System.IO;
+	using System.Linq;
+	using System.Security.Cryptography.X509Certificates;
+
+	internal static class CertificateLoader
```
Comment (mrts) [inline src/WebEid.AspNetCore.Example/Certificates/CertificateLoader.cs:8]: Duplicate code in web-eid-authtoken-validation-dotnet. Can/should this be exposed in web-eid-authtoken-validation-dotnet?

File: src/WebEid.AspNetCore.Example.Tests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace WebEid.AspNetCore.Example.Tests
+{
+	public class Tests
```
Comment (mrts) [inline src/WebEid.AspNetCore.Example.Tests/UnitTest1.cs:5]: What's with the test, do you plan to implement it?

**Review by mrts — CHANGES_REQUESTED** [review]

**Review by mrts — CHANGES_REQUESTED** [review]

File: src/WebEid.AspNetCore.Example/Certificates/CertificateLoader.cs (hunk 1 above)
Comment (agukasm) [reply src/WebEid.AspNetCore.Example/Certificates/CertificateLoader.cs]: In web-eid-authtoken-validation-dotnet certificates are loaded from resources, but here we read files

File: src/WebEid.AspNetCore.Example.Tests/UnitTest1.cs (hunk 2 above)
Comment (agukasm) [reply src/WebEid.AspNetCore.Example.Tests/UnitTest1.cs]: Nope :)

**Review by mrts — APPROVED** [review]

**Pull request merged**